style.css

```diff
@@ -6,4 +6,97 @@
     --blue: #006fc6;
     --black: #333;
     --white: #fff;
+}
+
+body {
+    background: var(--bg-color);
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+
+}
+
+/* nav bar */
+
+.nav {
+    display: flex;
```

Comment:
You are duplicating some rules a couple of times. Let's create utility classes for them.

For example, you can have a class named `flex` that adds `display: flex`. Another one `justify-content-between` that adds `justify-content: space-between`. Then we use these classes and get rid of duplicate code.